.circleci/config.yml

```diff
@@ -1739,6 +1743,54 @@ jobs:
                                 --acl public-read
                             from: ./release/.tmp/<< pipeline.parameters.graviteeio_version >>/dist
                             to: "s3://gravitee-releases-downloads/graviteeio-apim"
+    changelog_apim:
+      docker:
+        - image: groovy:jdk11
+          user: root
+      resource_class: small
+      environment:
+        MILESTONE_VERSION: << pipeline.parameters.gio_milestone_version >>
+      steps:
+        - checkout
+        - keeper/env-export:
+            secret-url: keeper://IZd-yvsMopfQEa_0j1SDvg/field/login
+            var-name: GIT_USER_NAME
+        - keeper/env-export:
+            secret-url: keeper://IZd-yvsMopfQEa_0j1SDvg/custom_field/email
+            var-name: GIT_USER_EMAIL
+        - keeper/env-export:
+            secret-url: keeper://TIlcGPFq4rN5GvgnZb9hng/field/password
+            var-name: GITHUB_TOKEN
+        - add_ssh_keys:
+            fingerprints:
+              - "ac:88:23:8f:c6:0f:7d:f0:fc:df:73:20:34:56:02:6c"
+        - run:
+            name: Git config
+            command: |
+              apt-get -y update
+              apt-get -y install git
+              git config --global user.name "${GIT_USER_NAME}"
+              git config --global user.email "${GIT_USER_EMAIL}"
```

Comment:
do we have a way to cache the package we install somewhere to avoid redownloading them ?

Reply:
Not sur it's possible to do what you expect unfortunaltly.